devices/src/lib.rs

```diff
@@ -59,4 +59,6 @@ pub enum Error {
     },
     IoError(io::Error),
     NoAvailBuffers,
+    MalformedDescriptor,
```

Comment:
We talked about this commit, the changes aren't significant on their own so I think it would be better squashed.